NightscoutServiceKit/OTPManager.swift

```diff
@@ -89,30 +89,32 @@ public class OTPManager {
         }
     }
     
```

Comment:
The validation of the OTP was refactored in this class to allow for better error messaging.